README.md

```diff
@@ -35,13 +35,31 @@ Once you have installed and set up the requirements, there are three steps to de
     This will will create and SSH key and various passwords that will be used for accessing the AWS
     servers and configuring XNAT.
 
+#### AppStream
+
+
+The AppStream image is only created if the `create_appstream` variable is set to `true`. When
+running `terraform apply`, the user will be prompted to enter a value for this variable.
```

Comment:
I think the user is only prompted if `create_appstream` is not defined in `terraform.tfvars` (and same with any variable that has no default and is not provided by the user). It might be better to add a default [here](https://github.com/UCL-MIRSG/xnat-aws/blob/a9ac92e178a1522ad181aa78477b85de7a297116/provision/vars.tf#L95-L98), but still worth mentioning appstream under this heading